content/docs/releasing.md

```diff
@@ -2,10 +2,14 @@
 title: "Releasing a new Open Source project"
 description: "Releasing a new Open Source project"
 draft: false
+images:
+    - /images/docs/about-white.svg
 ---
 
 Here you can see the steps that describe how Saucers can release a new open source project. This is a simple process, and if you need help going through it the Open Source team is here to give you a hand.
 
+{{< figure src="/images/docs/about-white.svg" class="img-responsive text-center" >}}
```

Comment:
I suggest to move the graphic to the bottom of the page. It might be more useful to sum things up with it.

Reply:
This image was changed for 2 different ones, specifically for contributions types and releasing line